pkgs/top-level/all-packages.nix

```diff
@@ -3287,7 +3287,13 @@ with pkgs;
 
   cinny = callPackage ../applications/networking/instant-messengers/cinny { stdenv = stdenvNoCC; };
 
-  cinny-desktop = callPackage ../applications/networking/instant-messengers/cinny-desktop { };
+  cinny-desktop = import ../applications/networking/instant-messengers/cinny-desktop {
+    inherit
+      lib
+      stdenvNoCC
+      callPackage
+      fetchurl;
+  };
```

Comment:
```suggestion
  cinny-desktop = callPackage ../applications/networking/instant-messengers/cinny-desktop { };
```

Reply:
That's done to avoid a double `callPackage` (since `default.nix` calls the platform agnostic derivations with it), which can make overrides a pain IIRC.